main.py

```diff
@@ -200,6 +204,85 @@ def randomchat(bot, update):
         bot.sendMessage(update.message.chat_id, text=reply)
 
 
+def estanoche(bot, update):
+    # Initalize coordinates
+    lan = None
+    lat = None
+
+    # TODO pass location as argument an resolve coordinates
+
+    if not lan or not lat:
+        # Yeah, Robledo de Chavela coordinates by default
+        lat = 40.498333
+        lon = -4.238889
+
+    # Building URL to query the service
+    url_service = 'http://202.127.24.18/bin/astro.php?lon='
```

Comment:
Requests has a nice option "params" that accepts a dict with the keys and the values to be passed to the url and does all the "heavy lifting" and url encoding for you. Look at the other commands and see how it works.